src/System.ComponentModel.Composition.Registration/ref/Configurations.props

```diff
@@ -1,8 +1,7 @@
 <Project>
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
```

Comment:
You are dropping netstandard configuration here which sh ould be a breaking change. Is this a new assembly? If not then I don't think you want to remove a configuration without harvesting from the previous package.

Reply:
It is a new assembly this release, but there are other side effects of dropping this.  You've eliminated the one reference assembly that lets folks target netstandard and run on both desktop and core.  Something to consider.  Granted it looks like before you were only shipping the notsupported assembly in this package... probably a bug in the package authoring.

Reply:
The package settings before were only temporary until we had netstandard2.1 support. I believe targeting NS21 here is the right path. The pkgproj also marks netfx as inbox.

Reply:
I don't think we should produce netstandard2.0 configurations for new packages if the API requires netstandard2.1. I get your point that this would be handy if you cross-compile for net4x and netstandard2.1 but I don't know yet if that's a common pattern. We could make adjustments here if necessary in the future.